Dockerfile

```
@@ -1,16 +1,17 @@
 FROM cm2network/steamcmd:root
 LABEL maintainer="thijs@loef.dev" \
-      name="thijsvanloef/palworld-server-docker" \
-      github="https://github.com/thijsvanloef/palworld-server-docker" \
-      dockerhub="https://hub.docker.com/r/thijsvanloef/palworld-server-docker" \
-      org.opencontainers.image.authors="Thijs van Loef" \
-      org.opencontainers.image.source="https://github.com/thijsvanloef/palworld-server-docker"
+    name="thijsvanloef/palworld-server-docker" \
+    github="https://github.com/thijsvanloef/palworld-server-docker" \
+    dockerhub="https://hub.docker.com/r/thijsvanloef/palworld-server-docker" \
+    org.opencontainers.image.authors="Thijs van Loef" \
+    org.opencontainers.image.source="https://github.com/thijsvanloef/palworld-server-docker"
```

```suggestion
      name="thijsvanloef/palworld-server-docker" \
      github="https://github.com/thijsvanloef/palworld-server-docker" \
      dockerhub="https://hub.docker.com/r/thijsvanloef/palworld-server-docker" \
      org.opencontainers.image.authors="Thijs van Loef" \
      org.opencontainers.image.source="https://github.com/thijsvanloef/palworld-server-docker"
```

While your changes here are meant to meet the 4 spaces I said earlier, the purpose for these extra spaces are so it's more easily readable as a continuation of LABEL. 
I would change this back so you're not listed in git blame.

---

Understood, it was just to get my linter to stop yelling at me, can be changed back.